app/screens/settings-screen/account-limits-screen.tsx

```diff
@@ -111,136 +100,77 @@ export const AccountLimitsScreen = () => {
-                        0,
-                      )} ${LL.AccountLimitsScreen.perWeek()}`}
-                    </Text>
-                  </View>
-                </View>
-                <View style={styles.divider} />
-                {txType === WalletCurrency.USD ? (
-                  <View style={styles.content}>
-                    <Text adjustsFontSizeToFit style={styles.valueFieldType}>
-                      {LL.AccountLimitsScreen.convertToStablesat()}
-                    </Text>
-                    <View style={styles.contentTextBox}>
-                      <Text adjustsFontSizeToFit style={styles.valueRemaining}>
-                        {`${usdAmountDisplay(
-                          Number(convertLimits?.DailyAccountLimit.remainingLimit),
-                          0,
-                        )} ${LL.AccountLimitsScreen.remaining().toLocaleLowerCase()}`}
-                      </Text>
-                      <Text adjustsFontSizeToFit style={styles.valueTotal}>
-                        {`${usdAmountDisplay(
-                          Number(convertLimits?.DailyAccountLimit.totalLimit),
-                          0,
-                        )} ${LL.AccountLimitsScreen.perDay()}`}
-                      </Text>
-                    </View>
-                    <View style={styles.contentTextBox}>
-                      <Text adjustsFontSizeToFit style={styles.valueRemaining}>
-                        {`${usdAmountDisplay(
-                          Number(convertLimits?.WeeklyAccountLimit.remainingLimit),
-                          0,
-                        )} ${LL.AccountLimitsScreen.remaining().toLocaleLowerCase()}`}
-                      </Text>
-                      <Text adjustsFontSizeToFit style={styles.valueTotal}>
-                        {`${usdAmountDisplay(
-                          Number(convertLimits?.WeeklyAccountLimit.totalLimit),
-                          0,
-                        )} ${LL.AccountLimitsScreen.perWeek()}`}
-                      </Text>
-                    </View>
-                  </View>
-                ) : null}
-              </View>
-            )
+    <Screen style={styles.container}>
+      <View style={styles.container}>
+        <View style={styles.limitWrapper}>
+          <Text adjustsFontSizeToFit style={styles.valueFieldType}>
+            {LL.AccountLimitsScreen.receive()}
+          </Text>
+          <View style={styles.content}>
+            <View style={styles.contentTextBox}>
+              <Text adjustsFontSizeToFit style={styles.valueRemaining}>
+                {LL.AccountLimitsScreen.unlimited()}
+              </Text>
+            </View>
+          </View>
+        </View>
+
+        <View style={styles.divider}></View>
+
+        <View style={styles.limitWrapper}>
+          <Text adjustsFontSizeToFit style={styles.valueFieldType}>
+            {LL.AccountLimitsScreen.withdraw()}
+          </Text>
+          {limits?.withdrawal.map((data, index: number) => {
```

Comment:
You should check with @vindard if there will be any guarantee that they are sorted from shortest to longest duration. Otherwise we will have to find a way to sort it on the frontend.

Reply:
waiting on a reply from him but it seems so from here src/graphql/types/object/account-limits.ts

Reply:
I think that even if the order is coming back from the API in the right way at the moment we should probably have code in the front end which orders it anyway.  If someone changes the order of an array on the backend they wouldn't expect that to change the rendering on the mobile app.

Reply:
Added this as a consideration under `point 3.` here: https://github.com/GaloyMoney/galoy/pull/1953#issuecomment-1357689726